pyface/tests/test_color.py

```diff
@@ -67,6 +67,14 @@ def test_from_str_hex(self):
         color = Color.from_str('#663399ff')
         self.assertEqual(color.rgba, (0.4, 0.2, 0.6, 1.0))
 
+    def test_from_str_extra_argument(self):
+        color = Color.from_str('#663399', alpha=0.5)
+        self.assertEqual(color.rgba, (0.4, 0.2, 0.6, 0.5))
+
+    def test_from_str_duplicate_argument(self):
+        self.assertRaises(TypeError):
```

Comment:
```suggestion
        with self.assertRaises(TypeError):
```

Reply:
This is what I get for trying to code when kids are wanting my attention 😞